presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -129,6 +129,16 @@ Memory Management Properties
     memory reservation across the cluster. The value of ``total-reservation-on-blocked-nodes``
     configures a policy that kills the query using the most memory on the workers that are out of memory (blocked).
 
+``driver.max-page-partitioning-buffer-count``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
You need more ^ to match the length of the previous line, I believe.
You can this the change by
```
cd presto-docs
make clean
make html
```
and take a look at the generated html in the target folder.